libpd_wrapper/z_libpd.h

```diff
@@ -18,97 +19,416 @@ extern "C"
 
 #include "m_pd.h"
 
+/* initializing pd */
+
+/// initialize libpd, it is safe to call this more than once
+/// returns 0 on success or -1 if libpd was already initialized
+/// note: sets SIGFPE handler to keep bad pd patches from crashing due to divide
+///       by 0, set any custom handling after calling this function
 EXTERN int libpd_init(void);
+
+/// clear the libpd search path for abstractions and externals
```

Comment:
s/clear/clears/ for consistency

This applies here and below.

Reply:
I'd need to change almost every entry. So far the convention is the function will "do" something and it then "returns" some value. Maybe it looks sloppy. It could changed to "return" instead which would then make it trivial to convert the commenting into real Doxygen, if someone wants to.

Reply:
Ok, I've done a little refactoring so the main description for getters is "get ..." or "check if..." then "returns" is only used when the return range is important/non-obvious.